script/openqa-bootstrap-container

```diff
@@ -47,13 +54,23 @@ DeviceAllow=block-blkext rw
 WantedBy=machines.target
 EOF
 
-mkdir $CONTANER_PATH
-zypper -n --root $CONTANER_PATH addrepo $DEFAULT_REPO defaultrepo
-zypper -n --root $CONTANER_PATH --gpg-auto-import-keys refresh
-zypper -n --root $CONTANER_PATH install --no-recommends -ly $PKGS_TO_INSTALL
+if [ ! -d $CONTAINER_PATH ] ; then
+	mkdir -p $CONTAINER_PATH
+	zypper -n --root $CONTAINER_PATH addrepo $DEFAULT_REPO defaultrepo
+	zypper -n --root $CONTAINER_PATH --gpg-auto-import-keys refresh
+	# There are non-fatal errors when zyppering inside chroot, so ignoring
+	set +e
+	zypper -n --root $CONTAINER_PATH install --no-recommends -ly $PKGS_TO_INSTALL
+	set -e
+else
+	echo Container path $CONTAINER_PATH already exists, stopping here. Please clean manually and rerun.
+	exit 1
+fi
 
 systemctl daemon-reload
 systemctl start systemd-nspawn-openqa@$CONTAINER_NAME
+# Wait a bit to avoid 'Failed to create bus connection: Protocol error'
+sleep 1
```

Comment:
Isn't the next while-loop doing exactly that?

Reply:
Theoretically yes, but apparently too early systemd-run may result in the error message mentioned.

Reply:
Hm, about we just pipe the errors from `systemd-run` into /dev/null when we expect them but do not want to see them?

@asdil12 ideas?

Reply:
> Hm, about we just pipe the errors from `systemd-run` into /dev/null when we expect them but do not want to see them?

my english parser fails here.

Reply:
```
# ignore expected error report about 'Failed to create bus connection: Protocol error'
systemd-run … >& /dev/null
```

Reply:
If I use for example:
`while ! timeout -s9 2 systemd-run -qPM $CONTAINER_NAME /bin/bash -c whoami >& /dev/null ; do systemctl restart systemd-nspawn-openqa@$CONTAINER_NAME.service ; sleep 3 ; done`
The original error is not seen but the systemctl restart fails and the script exists. If I add eg || true to the systemctl restart, like now committed, it eventually works but does two of the 'sleep 3's which is more waiting than the simple 'sleep 1'.

Reply:
Well, both not optimal but let's go for not confusing the user with false errors :) So I guess the way you did is fine. Thanks.